### PR TITLE
sbt 0.13.16

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

**Comment by sfergusonvolute** [comment]
Downloading, installing and updating sbt is causing issues for me today with version 0.13.16. It worked fine on Monday. Please advise.
/usr/share/sbt/bin/sbt-launch-lib.bash: line 58:    57 Killed                  "$@"
The command '/bin/sh -c sbt update' returned a non-zero code: 137
-------------------------------------------------------------------------------------------------

[warn] No sbt.version set in project/build.properties, base directory: /
/usr/share/sbt/bin/sbt-launch-lib.bash: line 58:   405 Killed                  "$@"
dpkg: error processing package sbt (--configure):
 subprocess installed post-installation script returned error exit status 137

**Comment by eed3si9n** [comment]
@sfergusonvolute Please open a separate issue following this guideline - http://www.scala-sbt.org/community.html#how-can-I-help

**Review by eed3si9n — CHANGES_REQUESTED** [review]
This looks legitimate error, but I'm not sure what's causing this:

```
[error] /home/travis/build/sbt/sbt/sbt-maven-resolver/src/test/scala/MavenResolutionSpec.scala:4: object specs2 is not a member of package org
[error] import org.specs2._
[error]            ^
[error] missing or invalid dependency detected while loading class file 'BaseIvySpecification.class'.
[error] Could not access type Specification in value org.specs2,
[error] because it (or its dependencies) are missing. Check your build definition for
[error] missing or conflicting dependencies. (Re-run with `-Ylog-classpath` to see the problematic classpath.)
[error] A full rebuild may help if 'BaseIvySpecification.class' was compiled against an incompatible version of org.specs2.
```

**Comment by dwijnand** [comment]
Indeed I gotta figure out how the hell upgrading sbt version is causing that error.

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=pull/3408/head&deploy_to_bintray=false">this link.</A>

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=pull/3408/head&deploy_to_bintray=false">this link.</A>

**Comment by typesafe-tools** [comment]
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=pull/3408/head&deploy_to_bintray=false">this link.</A>

File: build.sbt [hunk 1]
```diff
@@ -487,7 +487,7 @@ lazy val sbtProj = (project in sbtPath).
 lazy val mavenResolverPluginProj = (project in file("sbt-maven-resolver")).
   dependsOn(sbtProj, ivyProj % "test->test").
   settings(
-    baseSettings,
+    testedBaseSettings,
```
Comment (eed3si9n) [inline build.sbt:490]: So the root problem here was that this subproject was genuinely missing library dependencies for tests?

**Review by eed3si9n — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (dwijnand) [reply build.sbt:490]: No, transitively it should already have specs2. But Ivy evicts the dependency on 0.13.16-SNAPSHOT artefacts (from `version`) in favour of 0.13.16 artefacts (from `sbt.version`).. believe it or not..

Comment (eed3si9n) [reply build.sbt:490]: lol. I am guessing this is yet another Ivy-Maven impedance mismatch since Maven doesn't make Tests public.

Comment (eed3si9n) [reply build.sbt:490]: Also shouldn't we bump the version number if that a cause of so much trouble?

Comment (dwijnand) [reply build.sbt:490]: in 0.13 everything's Ivy, so it's an impedance mismatch between what Ivy thinks of version strings and which version strings we pick for sbt..

**Pull request merged**